new/LIT7085Papur.xml

```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7085Papur" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ ማዕሠረ፡ አጋንንት፡ ጰጱር፡ ጵውያካኤል፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ṣalot baʾǝnta māʿśara ʾagānǝnt ṗaṗur ṗǝwyākāʾel</title>
+                <title xml:lang="en" corresp="#t1">Prayer for binding demons ṗaṗur ṗǝwyākāʾel</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Magic prayer for binding demons with the help of magic words (<foreign xml:lang="gez">ʾasmāt</foreign> or 
+                    <foreign xml:lang="gez">qālāt</foreign>). 
+                    The word <foreign xml:lang="gez">ṗaṗur</foreign> is attested in the form of <foreign xml:lang="gez">ṗaṗar</foreign>, 
+                    <foreign xml:lang="gez">ṗaṗurlāmā</foreign> (also: <foreign xml:lang="gez">papurlāmā</foreign>) and
+                    <foreign xml:lang="gez">ṗaṗurālābā</foreign> in different prayers in <ref type="mss" corresp="BAVet128">ms. Vat. 128</ref>, each one
```

Comment:
Is it not better to leave the manuscript reference element empty (without "ms. Vat. 128"), so that it the full shelfmark is rendered?

I'm asking out of Unwissen.

Reply:
Yes, it is better in this case.